python/cugraph/cugraph/tests/dask/test_mg_renumber.py

```diff
@@ -207,7 +212,8 @@ def test_directed_graph_renumber_false(renumber, dask_client):
     is_single_gpu(), reason="skipping MG testing on Single GPU system"
 )
 @pytest.mark.parametrize("renumber", [False])
-def test_multi_directed_graph_renumber_false(renumber, dask_client):
+@pytest.mark.parametrize("directed", IS_DIRECTED)
+def test_multi_directed_graph_renumber_false(renumber, dask_client, directed):
```

Comment:
This test name implies it's only testing directed graphs.  I'm curious if this test should fail for undirected then?